tests/lib/rules/comma-dangle.js

```diff
@@ -271,7 +271,7 @@ ruleTester.run("comma-dangle", rule, {
             options: ["always-multiline"],
         },
         {
-            code: "foo(\na,\mb\m)",
```

Comment:
Yeah, I was not sure about this one, as the test above this one is also testing `"foo(\na,\nb\n)"`. I might as well just delete this one?